src/template.rs

```diff
@@ -0,0 +1,30 @@
+use home::home_dir;
+
+use std::env;
+use std::fs;
+use std::io;
+use std::path::PathBuf;
+
+#[derive(Debug)]
+pub enum Error {
+    CouldNotGetHomeDirectory,
+    Other(io::Error),
+}
```

Comment:
In the rest of Bonnie, we generate string errors at the site of error and propagate them up to `main`. Could you please remove this `enum` and switch to that model? We will eventually switch to `error_chain`, but for now I'd like to keep things in the same format. This will need changes throughout your code.